packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -393,9 +393,7 @@ export function TransferPanel() {
       try {
         await switchNetworkAsync?.(switchTargetChainId)
       } catch (e) {
-        if (isUserRejectedError(e)) {
-          return
-        }
+        Sentry.captureException(e)
```

Comment:
do we still want to capture exception here?

Reply:
I think we should. It won't capture anything if it's an user rejection